include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -1136,7 +1141,7 @@ struct __brick_move<_Tag, _ExecutionPolicy, ::std::enable_if_t<__is_host_dispatc
     }
 };
 
-template <class _Tag, typename _ExecutionPolicy, typename = ::std::enable_if_t<__is_host_dispatch_tag_v<_Tag>>>
+template <class _Tag, typename = std::enable_if_t<__is_host_dispatch_tag_v<_Tag>>>
```

Comment:
It looks that `__brick_move_destroy` is used only for the host  host code.
And it called with `__parallel_tag` only.
So, we can write `struct __brick_move_destroy` w/o any template parameters.

Reply:
You are correct. But this point doesn't correlated with this PR goal, so I propose create separate issue.

Reply:
Issue created: https://github.com/uxlfoundation/oneDPL/issues/2179